backend/dataall/modules/datasets/handlers/glue_column_handler.py

```diff
@@ -0,0 +1,141 @@
+            except glue_client.exceptions.ClientError as e:
+                log.error(
+                    f'Failed to get table aws://{dataset_table.AWSAccountId}'
+                    f'//{dataset_table.GlueDatabaseName}'
+                    f'//{dataset_table.name} due to: '
+                    f'{e}'
+                )
+            DatasetTableService.sync_table_columns(
+                session, dataset_table, glue_table['Table']
+            )
+        return True
+
+    @staticmethod
+    @Worker.handler('glue.table.update_column')
+    def update_table_columns(engine, task: models.Task):
+        with engine.scoped_session() as session:
+            column: DatasetTableColumn = session.query(
+                DatasetTableColumn
+            ).get(task.targetUri)
+            table: models.DatasetTable = session.query(models.DatasetTable).get(
+                column.tableUri
+            )
+            try:
+                aws_session = SessionHelper.remote_session(table.AWSAccountId)
+
+                DatasetColumnGlueHandler.grant_pivot_role_all_table_permissions(aws_session, table)
+
+                glue_client = aws_session.client('glue', region_name=table.region)
+
+                original_table = glue_client.get_table(
+                    CatalogId=table.AWSAccountId,
+                    DatabaseName=table.GlueDatabaseName,
+                    Name=table.name,
+                )
+                updated_table = {
+                    k: v
+                    for k, v in original_table['Table'].items()
+                    if k not in [
+                        'CatalogId',
+                        'VersionId',
+                        'DatabaseName',
+                        'CreateTime',
+                        'UpdateTime',
+                        'CreatedBy',
+                        'IsRegisteredWithLakeFormation',
+                    ]
+                }
+                all_columns = updated_table.get('StorageDescriptor', {}).get(
+                    'Columns', []
+                ) + updated_table.get('PartitionKeys', [])
+                for col in all_columns:
+                    if col['Name'] == column.name:
+                        col['Comment'] = column.description
+                        log.info(
+                            f'Found column {column.name} adding description {column.description}'
+                        )
+                        response = glue_client.update_table(
+                            DatabaseName=table.GlueDatabaseName,
+                            TableInput=updated_table,
+                        )
+                        log.info(
+                            f'Column {column.name} updated successfully: {response}'
+                        )
+                return True
+
+            except ClientError as e:
+                log.error(
+                    f'Failed to update table column {column.name} description: {e}'
+                )
+                raise e
+
```

Comment:
This method does not make much sense here. we have the glue_table and the glue_column handlers and this is a function is a lf_table function that is inside the glue_column handler.

In total we have 2 functionalities:
- glue (tables and columns) schema handler
- lake formation permissions handler

Reply:
Agree! I was planning to split the handlers code into two pieces: `handlers` and `clients`. Handlers is the code that's need for  `@Worker.handler` and `clients` will contain API to make calls to AWS. I wanted to do it in the refactoring PR, but since you pointed it out, I will do it here at least for LF